website/src/content/components/application-shell.mdx

```diff
@@ -42,7 +42,7 @@ const EntryPoint = () => (
 | `applicationMessages`            | `object` or `func` |           ✅            | -       | This is either an object containing all the translated messages, grouped by locale (`{ "en": { "Welcome": "Welcome" }, "de": { "Welcome": "Wilkommen" }}`), or a function that returns a Promise that resolves to such an object. The function is called with a `locale` parameter. See [Using the messages in the application](/development/translations#using-the-messages-in-the-application).                                                                                                                                                                |
 | `environment`                  | `object`           |           ✅            | -       | The object containing the `env.json`. By default this is available in `window.app`. See [Runtime configuration](/deployment/runtime-configuration#envjson).                                                                                                                                                                                                                                                                                                                                                       |
 | `render`                         | `func`             |           ✅            | -       | The render function is called when the `<ApplicationShell>` is ready to render the actual application. This is the case when the required data (user, project) has been fetched and the application context has been initialized.                                                                                                                                                                                                                                    |
-| `trackingEventWhitelist`         | `object`           |                       | -       | An optional object containing a map of tracking events.                                                                                                                                                                                                                                        |
+| `trackingEventList`         | `object`           |                       | -       | An optional object containing a map of tracking events.                                                                                                                                                                                                                                        |
```

Comment:
instead of removing it entirely, I would:

1. keep it and mark it deprecated. perhaps announce a date of when we will remove it completely from the code base e.g this https://github.com/commercetools/merchant-center-application-kit/pull/1614/files#diff-296eae346ba59229997210b8ff8ab651R557
2. document to use the new name instead.

Then we can eventually remove the old name

Reply:
I thought about that but we only use this feature internally, so I wouldn't bother document it properly.

Reply:
aahaa, I was under the impression otherwise.
so help me understand the strategy here:
are we going to refactor this off from MC-FE then remove the old naming entirely in the next step? #guessWork

Reply:
Yes, I assumed so too.

Reply:
Yep, more or less

Reply:
all right. thanks